src/modules/70-pipeline/components/PipelineStudio/PipelineContext/PipelineContext.tsx

```
@@ -76,6 +76,7 @@ interface PipelineInfoConfigWithGitDetails extends PipelineInfoConfig {
 }
 
 const logger = loggerFor(ModuleName.CD)
+const DBNotFoundErrorMessage = 'There was no DB found'
```

We are logging the same error message for both the cases --> if error occurs while getting entity from db or deleting it and ignoring the error got i catch -->
So should we log that also , so that it would be easier in debugging and going through the logs. 

Not sure if this should be done, just a thought.

---

Logger being used accepts the string, but the error encountered contains an object with no fixed field, so added generic message for the same.